packages/batch-submitter/src/batch-submitter/state-batch-submitter.ts

```diff
@@ -98,7 +98,7 @@ export class StateBatchSubmitter extends BatchSubmitter {
       'appendStateBatch',
       [batch, startBlock]
     )
-    if (tx.length < this.minTxSize) {
```

Comment:
I think that multiplying when setting `this.minTxSize` is safer, in case the variable ends up getting used elsewhere in the codebase

Reply:
Ah I like that! I'll change it -- one moment